src/integrations/eventorigin.ts

```diff
@@ -15,8 +15,8 @@ export class EventOrigin implements Integration {
   /**
    * @inheritDoc
    */
-  public setupOnce(addGlobalEventProcessor: (e: EventProcessor) => void): void {
-    addGlobalEventProcessor(event => {
```

Comment:
Just note, the callback is still `addGlobvalEventProcessor`. 

https://github.com/getsentry/sentry-javascript/blob/a7097d9ba2a74b2cb323da0ef22988a383782ffb/packages/core/src/integration.ts#L134

Reply:
Would you rather rollback this change and add the deprecated mark?

Reply:
I would rather rollback in this case because the global in the name is more accurate, I believe the lint won't complain here, as the callback in not imported like in the other integrations.